packages/webpack-plugin/test/e2e/projects/stylable-config-resolver-override/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+const { StylableWebpackPlugin } = require('@stylable/webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+/** @type {import('webpack').Configuration} */
+module.exports = {
+    mode: 'development',
+    context: __dirname,
+    devtool: 'source-map',
+    plugins: [new StylableWebpackPlugin(), new HtmlWebpackPlugin()],
```

Comment:
missing override/inherit config from the webpack->plugin->config

Reply:
added another test to verify this behavior